stat/int.go

```diff
@@ -0,0 +1,27 @@
+package stat // import "github.com/influxdata/influxdb/stat"
+
+import "sync"
+
+type Int struct {
+	mu    sync.RWMutex
```

Comment:
Your benchmark is benchmarking goroutine creation and waitgroup contention... `b.N` gets very large for fast operations, you created 5 million goroutines when you posted your results. There are better ways to test parallel access than 1 goroutine per `b.N`